hubspot/management/commands/configure_hubspot_bridge.py

```diff
@@ -87,7 +87,19 @@
                 "fieldType": "text",
             },
         ],
-    }
+    },
+    "contacts": {
+        "properties": [
+            {
+                "name": "highest_education",
+                "label": "Highest Education",
+                "description": "Highest education level",
+                "groupName": "contactinformation",
+                "type": "string",
+                "fieldType": "text",
+            }
+        ]
+    },
 }
 
 HUBSPOT_ECOMMERCE_SETTINGS = {
```

Comment:
Need to add `highest_education` here too, under `contactSyncSettings.properties`:
```
            {
                "propertyName": "highest_education",
                "targetHubspotProperty": "highest_education",
                "dataType": "NUMBER",  # (or "STRING")
            },
```

Reply:
added